mathesar/views/api.py

```diff
@@ -218,6 +225,14 @@ def list(self, request):
         return Response(get_non_default_database_keys())
 
 
+class DatabaseViewSet(viewsets.GenericViewSet, ListModelMixin, RetrieveModelMixin):
+    def get_queryset(self):
+        reflect_db_objects()
+        return Database.objects.all().order_by('-created_at')
```

Comment:
The `default` database should not be included since it's Mathesar's internal database, it's not managed by Mathesar.